apps/wallet-mobile/src/features/Scan/common/CodeScannerButton.tsx

```diff
@@ -1,17 +1,26 @@
+import {useTheme} from '@yoroi/theme'
 import * as React from 'react'
 import {TouchableOpacityProps} from 'react-native'
 
 import {Icon} from '../../../components'
 import {PressableIcon} from '../../../components/PressableIcon/PressableIcon'
-import {COLORS} from '../../../theme'
 
 const fallbackSize = 30
```

Comment:
@SorinC6 icons size should also be on theme, later if we apply breakpoints/scale, the icons will be fixed :/